nss-evening-cohort-13/Spinnovations
# Seller store

## Description
When you are looking at a single product, the user can click on a spinnovator's name to see all products added by that user.

## Related Issue
Resolves the ticket with title: Customer can click on a seller's name on a product page to view all products sold by that seller.

## How Can This Be Tested?
```git fetch origin seller-store```

Click on one of the items you have added. Click on your name to see all of the items you have added.

## Screenshots (if appropriate):
![single-seller](https://user-images.githubusercontent.com/63985074/117085035-5872d480-ad0e-11eb-9b25-23410ffeabee.PNG)

## Types of changes
